diff --git a/Makefile
index ac9d1a4..9e46a51 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 .SUFFIXES:
 
 DESTDIR=
-prefix=/usr/local
+prefix=/usr/local
 bindir=$(prefix)/bin
 mandir=$(prefix)/man
 man1dir=$(mandir)/man1
@@ -42,8 +42,8 @@ WARNINGS=-Wall \
          -Wwrite-strings -Wstrict-prototypes -Wmissing-prototypes \
          -Wnested-externs -Winline
 OPTIMIZE=-O3 -fno-strength-reduce
-CFLAGS=$(WARNINGS) $(OPTIMIZE)
-LDFLAGS=
+CFLAGS=$(WARNINGS) $(OPTIMIZE) -I/usr/local/include
+LDFLAGS=-L/usr/local/lib -lintl
 
 sources=getopt.c
 ifeq ($(LIBCGETOPT),0)
