diff --git a/xmlto.in b/xmlto.in
index aed5458..311ee37 100755
--- a/xmlto.in
+++ b/xmlto.in
@@ -225,7 +225,7 @@ VERBOSE=0
 export VERBOSE
 
 # Disable network entities
-XSLTOPTS="$XSLTOPTS --nonet"
+#XSLTOPTS="$XSLTOPTS --nonet"
 
 # The names parameter for the XSLT stylesheet
 XSLTPARAMS=""
diff --git a/Makefile.am b/Makefile.am
index 50fa279..6a2da62 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -68,7 +68,7 @@ EXTRA_DIST = xmlto.spec \
        doc/xmlif.xml \
        xmlto.mak
 
-GEN_MANPAGE = FORMAT_DIR=$(top_srcdir)/format $(BASH) ./xmlto -o $(@D) man $<
+GEN_MANPAGE = FORMAT_DIR=$(top_srcdir)/format $(BASH) ./xmlto --skip-validation -o $(@D) man $<
 man/man1/xmlto.1: doc/xmlto.xml ; $(GEN_MANPAGE)
 man/man1/xmlif.1: doc/xmlif.xml ; $(GEN_MANPAGE)
 
