diff --git a/xmlto.in b/xmlto.in
index aed5458..311ee37 100755
--- a/xmlto.in
+++ b/xmlto.in
@@ -225,7 +225,7 @@ VERBOSE=0
 export VERBOSE
 
 # Disable network entities
-XSLTOPTS="$XSLTOPTS --nonet"
+#XSLTOPTS="$XSLTOPTS --nonet"
 
 # The names parameter for the XSLT stylesheet
 XSLTPARAMS=""
--- a/Makefile.in
+++ b/Makefile.in
@@ -581,7 +581,7 @@
 	doc/xmlif.xml \
 	xmlto.mak
 
-GEN_MANPAGE = FORMAT_DIR=$(top_srcdir)/format $(BASH) ./xmlto --skip-validation -o --skip$(@D) man $<
+GEN_MANPAGE = FORMAT_DIR=$(top_srcdir)/format $(BASH) ./xmlto -o --skip$(@D) man $<
 TESTS_ENVIRONMENT = top_srcdir=$(top_srcdir) top_builddir=$(top_builddir)
 TESTS = xmlif/test/run-test
 CLEANFILES = $(dist_man1_MANS) \
